test/test_overrides.py

```diff
@@ -978,5 +978,17 @@ def test_iterator(self):
         self.assertIs(type(next(it)), SubTensor2)
         self.assertIs(type(next(it)), SubTensor2)
 
+
+class TestRNN(TestCase):
+    # Regression test for gh-55868
+    def test_rnn(self):
+        try:
+            model = torch.nn.RNN(10, 20, 2)
+            input = Wrapper(torch.randn(1, 5, 10))
+            model(input)
+        except Exception as e:
```

Comment:
do you expect some exception raises? If yes, check explicitly for the exception; otherwise why we need try-except?

Reply:
This can either pass or fail here, but this test needs re-writing, I've marked the PR as draft for now.